docs/cdn_troubleshooting.md

```diff
@@ -188,4 +205,22 @@ application before enabling this option in a production environment.
         }
     );
 </script>
-```
\ No newline at end of file
+```
+---
+## Route change timing is not recorded for my single page application
+
+### Configuring `routeChangeTimeout`
+The current mechanism to calculate a route change (virtual page load) is the difference between the latest completion time of a DOM mutation or an outgoing AJAX request and the user interaction that triggered the route change.
```

Comment:
Our goal in this doc is to provide guidance for a specific problem that we know application owners encounter. Are these issues that we know application owners are likely to encounter?

If "no", we can remove these sections -- the config docs are sufficient.

If "yes", the first sentence/paragraph should explicitly state what the problem is. State what the symptoms are, and possibly give some context about why they appear, but try to avoid implementation details. And of course, provide actionable steps to resolve the problem.